source/educators/references/resources_for_course_teams.rst

```diff
@@ -1,58 +1,58 @@
-**********************************
-Resources for edx.org Course Teams
-**********************************
+###################################
+Resources for Open edX Course Teams
+###################################
 
 .. tags:: educator, reference
 
 Course teams include faculty, instructional designers, course staff, discussion
 moderators, and others who contribute to the creation and delivery of courses
-on edx.org or edX Edge.
+on Open edX platform.
 
-======================================
-The edX Course Creator Series
-======================================
+**********************************
+The Open edX Course Creator Series
+**********************************
 
-The courses in the edX Course Creator Series provide foundational knowledge
-about using the edX platform to deliver educational experiences. These courses
-are available on edx.org.
+The courses in the Open edX Course Creator Series provide foundational knowledge
+about using the Open edX platform to deliver educational experiences. These courses
+are available on Open edX platform.
 
 .. contents::
  :local:
  :depth: 1
 
-edX101: Overview of Creating a Course
-**************************************
+Open edX101: Overview of Creating a Course
```

Comment:
```suggestion
Open edX Demo Course
```

We no longer have the "edX101" course. Please change this section to reference the Demo course and explain it briefly (look at the about page on https://sandbox.openedx.org/courses/course-v1:OpenedX+01-2024+2024-1/about for more detail)